test/integration/apiserver.go

```diff
@@ -471,6 +472,31 @@ func offeringService(ctx context.Context, conn *grpc.ClientConn, managementClien
 			assert.Equal(t, expectedResult, offering)
 			return true, nil
 		}, offeringCtx.Done()))
+
+		// watch offerings
+		watchClient, err := client.Watch(offeringCtx, &apiserverv1.OfferingWatchRequest{
```

Comment:
testing wise I'd love seeing more events than pure `ADDED`. Maybe delte the offering to get the `DELETED` event type.

Reply:
I have added more events, for example, `Deleted` and `Modified` events.